Pod/Sources/OriginateRemoteCollection+Internal.h

```diff
@@ -0,0 +1,33 @@
+//
+//  OriginateRemoteCollection+Internal.h
+//  Pods
+//
+//  Created by Philip Kluz on 10/3/15.
+//
+//
+
+@import Foundation;
+
+typedef NS_ENUM(NSUInteger, OriginateRemoteCollectionState) {
+    OriginateRemoteCollectionStateNone,
+    OriginateRemoteCollectionStateIdle,
+    OriginateRemoteCollectionStateLoading
+};
+
+@interface OriginateRemoteCollection ()
+{
+    NSArray *_objects;
+}
+
+#pragma mark - Properties
+@property (nonatomic, assign, readwrite) OriginateRemoteCollectionState state;
+@property (nonatomic, strong, readwrite) NSError *lastError;
+@property (nonatomic, strong, readwrite) NSArray *objects;
```

Comment:
Since this is for internal use anyway, and you're using both `_objects` and `self.objects`, what do you think about removing property and sticking with the ivar?